### PR TITLE
update links

### PR DESCRIPTION
**Pull request by cdetrio**
Once EIPs are merged and have files, links should be updated to reference the files rather than the issues.

Also a couple of whitespace fixes.

### CODE REVIEW

**Comment by Souptacular** [comment]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ First review [EIP-1](EIPS/eip-1.md). Then clone the repository and add your EIP
...
 | ------------------------------------------------------- | ----------------------------------------------------------- | ----------------| ------------| --------|
-| [2](EIPS/eip-2.md)                               | Homestead Hard-fork Changes                                 | Vitalik Buterin | Core        | Final   |
+| [2](EIPS/eip-2.md)                                      | Homestead Hard-fork Changes                                 | Vitalik Buterin | Core        | Final   |
 | [6](EIPS/eip-6.md)                                      | Renaming Suicide Opcode                                     | Hudson Jameson  | Interface   | Final   |
 | [7](EIPS/eip-7.md)                                      | DELEGATECALL                                                | Vitalik Buterin | Core        | Final   |
 | [8](EIPS/eip-8.md)                                      | devp2p Forward Compatibility Requirements for Homestead     | Felix Lange     | Networking  | Final   |
 | [141](EIPS/eip-141.md)                                  | Designated invalid EVM instruction                          | Alex Beregsazszi| Core        | Final   |
```
Comment (axic) [inline README.md]: @hudson would you mind fixing the typo in my name here? :)

Comment (Souptacular) [inline README.md]: Blast! Name typos are the most difficult to detect :P

Comment (cdetrio) [inline README.md]: i got it

**Comment by Souptacular** [comment]
@axic is it good now? I think I caught all 2 instances of your name.

**Comment by axic** [comment]
Cool thanks! 🎉

**Pull request merged**